packages/airnode-utilities/src/caching/index.ts

```diff
@@ -0,0 +1,252 @@
+import { existsSync, mkdirSync, readdirSync, readFileSync, rmdirSync, rmSync, statSync, writeFileSync } from 'fs';
+import { spawn } from 'child_process';
+import { join } from 'path';
+import { logger } from '../logging';
+
+export const CACHE_BASE_PATH = `/tmp/airnode-cache`;
+export const CACHE_MAX_FILESYSTEM_AGE_MINUTES = 60;
+export const CACHE_MAX_FILES = 100_000; // (500 (cache size) * 1024 (KB) ) / 4 (KB, fs "cluster size" = 130k
```

Comment:
I am not sure I understand this comment, also 100K seems to me like too many files. 

Why is the cache size 500 ? Why `* 1024 / 4` ?

Reply:
Maybe @ 100K :D 
The Lambda `/tmp` directory is 500 MB in size. It uses a 4 KB sector/record size (different file systems call it different things, but it's the minimum file size).
500 MB * 1024 = KB, KB /4 gives you the maximum number of minimum-sized files, which works out to around 128 000 files.
I figured that 100k files was more than enough and gives us some headroom, so that's what I made the number, but we can make it 20k.
You'll probably tell me that I got the maths wrong @ Kibi vs Kilo :P but I tested the above limits too by running some test code on Lambda.

Reply:
Where did you get the information that minimum file size is 4 KB? That seems way too much for a minimum file size.

Reply:
4 KB is a common sector size for modern filesystems. I think found the size using `statSync` on a freshly created file, but I can test again some time.

It is also the standard minimum file size for Linux tmpfs based volumes.

Reply:
Thanks for the links! :)